test/e2e/specs/editor/various/behaviors.spec.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Testing behaviors functionality', () => {
+	const filename = '1024x768_e2e_test_image_size.jpeg';
+	const filepath = path.join( './test/e2e/assets', filename );
+
+	const createMedia = async ( { admin, requestUtils } ) => {
+		await admin.createNewPost();
+		const media = await requestUtils.uploadMedia( filepath );
+		return media;
+	};
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+		await requestUtils.deleteAllPosts();
+	} );
+
+	test.afterEach( async ( { requestUtils } ) => {
+		await requestUtils.deleteAllMedia();
+	} );
+
+	test( '`No Behaviors` should be the default as defined in the core theme.json', async ( {
+		admin,
+		editor,
+		requestUtils,
+		page,
+	} ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+		const media = await createMedia( { admin, requestUtils } );
+		await editor.insertBlock( {
+			name: 'core/image',
+			attributes: {
+				alt: filename,
+				id: media.id,
+				url: media.source_url,
+			},
+		} );
+
+		await page.getByRole( 'button', { name: 'Advanced' } ).click();
+		const select = page.getByLabel( 'Behavior' );
+
+		// By default, no behaviors should be selected.
+		await expect( select ).toHaveCount( 1 );
+		await expect( select ).toHaveValue( '' );
+
+		// By default, you should be able to select the Lightbox behavior.
+		const options = select.locator( 'option' );
+		await expect( options ).toHaveCount( 2 );
+	} );
+
+	test( 'Behaviors UI can be disabled in the `theme.json`', async ( {
+		admin,
+		editor,
+		requestUtils,
+		page,
+	} ) => {
+		// { "lightbox": true } is the default behavior setting, so we activate the
+		// `behaviors-ui-disabled` theme where it is disabled by default. Change if we change
+		// the default value in the core theme.json file.
+		await requestUtils.activateTheme( 'behaviors-ui-disabled' );
+		const media = await createMedia( { admin, requestUtils } );
+
+		await editor.insertBlock( {
+			name: 'core/image',
+			attributes: {
+				alt: filename,
+				id: media.id,
+				url: media.source_url,
+			},
+		} );
+
+		await page.getByRole( 'button', { name: 'Advanced' } ).click();
+
+		// No behaviors dropdown should be present.
+		await expect( page.getByLabel( 'Behavior' ) ).toHaveCount( 0 );
```

Comment:
A more readable and explicit assertion might be:

```suggestion
		await expect( locator ).toBeHidden();
```